.reuse/dep5

```diff
@@ -14,7 +14,3 @@ License: CC-BY-4.0
 Files: data/*
 Copyright:  The PyPSA-Earth and PyPSA-Eur Authors
 License: CC-BY-4.0
-
-Files: .github/*
-Copyright:  The PyPSA-Earth and PyPSA-Eur Authors
-License: CC0-1.0
```

Comment:
@juli-a-ko, after giving some thoughts to dep5, I fear that `.github/*` files should be included there. The reason is templates which are placed in `.github` directory. License headers make working with the templates difficult, which has caused need of [urgent changes](https://github.com/pypsa-meets-earth/pypsa-earth/pull/660/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3) 

If I understand properly, that mean we have to leave `.github/*` in `dep5`. Could you please double-check this logic and adjust the PR accordingly?